source/shared/cpp/ObjectModel/CaptionSource.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "pch.h"
+#include "BaseCardElement.h"
+#include "ElementParserRegistration.h"
+#include "Util.h"
+
+namespace AdaptiveCards
+{
+class CaptionSource
+{
+public:
+    CaptionSource();
+    CaptionSource(const CaptionSource&) = default;
+    CaptionSource(CaptionSource&&) = default;
+    CaptionSource& operator=(const CaptionSource&) = default;
+    CaptionSource& operator=(CaptionSource&&) = default;
+    virtual ~CaptionSource() = default;
```

Comment:
Do we need to define any of those?:) they're still **default** and compiler will do it for us:)

Reply:
Good call! Removed